src/MetaMorpho.sol

```diff
@@ -284,6 +285,18 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         }
     }
 
+    /// @notice Submits a forced market removal from the vault, losing all funds supplied to the market.
```

Comment:
```suggestion
    /// @notice Submits a forced market removal from the vault, eventually losing all funds supplied to the market.
```
Because if the market stops reverting, the funds might be recovered